dowp.bash

```diff
@@ -31,7 +31,7 @@ postPopper () {
   rm "${thisDir}/dowpPosts/post." "${thisDir}/dowpPosts/post.0" 2> /dev/null #Garbage file
   for fileName in `ls ${thisDir}/dowpPosts/p*` ; do
     postDateTime=`grep "<wp:post_date>" ${fileName} | sed -e 's/<wp:post_date>//' | sed -e 's/<\/wp:post_date>//' | sed -e 's/\-/\//g'`
-    postYear=`echo ${postDateTime} | cut -d"/" -f1`
+    postYear=`echo ${postDateTime} | cut -d"/" -f1 | egrep -o '[[:digit:]]{4}' | head -n1`
```

Comment:
Here is what the year was looking like before this change:
<img width="454" alt="screen shot 2017-02-26 at 3 56 21 pm" src="https://cloud.githubusercontent.com/assets/5497885/23344670/ad53d96c-fc3d-11e6-9ca7-aa187242e213.png">
<img width="377" alt="screen shot 2017-02-26 at 3 55 02 pm" src="https://cloud.githubusercontent.com/assets/5497885/23344671/ad54034c-fc3d-11e6-9f54-3b668f7a56bc.png">

Now it looks like this:

<img width="155" alt="screen shot 2017-02-26 at 4 01 32 pm" src="https://cloud.githubusercontent.com/assets/5497885/23344672/ad54d15a-fc3d-11e6-88ed-0ec4ce84b5c1.png">

Which renders correctly in the Day One app:
<img width="91" alt="screen shot 2017-02-26 at 4 04 01 pm" src="https://cloud.githubusercontent.com/assets/5497885/23344669/ad52eb56-fc3d-11e6-8047-0b9aa95ef291.png">

Without this change the year gets the default year which is the current year.